apis/core/v1beta1/openstackcontrolplane_types.go

```diff
@@ -65,6 +67,14 @@ type OpenStackControlPlaneSpec struct {
 	// +kubebuilder:validation:Optional
 	// RabbitmqTemplate - Overrides to use when creating the Rabbitmq cluster
 	RabbitmqTemplate rabbitmqv1.RabbitmqClusterSpec `json:"rabbitmqTemplate,omitempty"`
+
+	// +kubebuilder:validation:Optional
+	// AnsibleTemplate - Overrides to use when creating the Ansibleee job
+	AnsibleTemplate ansibleeev1.AnsibleEESpec `json:"ansibleTemplate,omitempty"`
```

Comment:
I need a bit more info about the need for this. Why do we need an AnsibleTemplate within the OpenStackControlplane resource? I thought we only needed Ansible for the Dataplane deployment.

Reply:
See comment [1] as a general answer to your question, which makes sense.
If the goal is triggering an `AnsibleEE` job from the individual operators (that actually build the info for the `OpenStackDataPlane CR`, then an ansible-template is out of scope here.

[1] https://github.com/openstack-k8s-operators/openstack-operator/pull/38#discussion_r1018808545